src/server/api/controllers/users.controller.js

```diff
@@ -1,10 +1,11 @@
-/* TODO: This is an example controller to illustrate a server side controller.
-Can be deleted as soon as the first real controller is added. */
+/*  This is the controller for cvs */
 
 const knex = require('../../config/db');
+
 const Error = require('../lib/utils/http-error');
 
 const moment = require('moment-timezone');
+const { request } = require('express');
```

Comment:
This remove be removed :) as it is not used

Reply:
yap true, let me remove it